.env-example

```diff
@@ -5,4 +5,5 @@ export NODE_ENV="LOCAL_DEV"
 export COOKIE_DOMAIN="localhost"
 export AUTH_TOKEN_EXPIRY_DURATION_IN_MILLISECONDS=3600000
 export JWT_SECRET=mysecretblah
-export FRONTEND_URL='http://localhost:8081'
\ No newline at end of file
+export FRONTEND_URL='http://localhost:8081'
+export MUTEX_TABLE_NAME=""
```

Comment:
Do we need 
```
export AWS_ACCESS_KEY_ID=""
export AWS_SECRET_ACCESS_KEY=""
``` 
to test dynamodb from a local server?

Reply:
Resolved in 50da021